crt_portal/cts_forms/views.py

```diff
@@ -155,6 +153,33 @@ def setup_filter_parameters(report, querydict):
     return output
 
 
+def get_requested_ids(return_url_args):
+    return_url_querydict = QueryDict(return_url_args[1:])
+    activity = return_url_querydict.get('activity', None)
+    if activity:
+        requested_query = reconstruct_activity_query(return_url_args)
+        return list(map(int, requested_query.values_list('target_object_id', flat=True)))
+    else:
+        requested_query = reconstruct_query(return_url_args)
+        return list(requested_query.values_list('id', flat=True))
```

Comment:
```suggestion
    requested_query = reconstruct_query(return_url_args)
    return list(requested_query.values_list('id', flat=True))
```

supernit: unecessary else after if + return; since this is your base case (standard report list), imo it shouldn't be indented